src/ray/protobuf/gcs.proto

```diff
@@ -690,4 +665,32 @@ message PlacementGroupTableData {
   // effectively when the placement group has been "scheduled".
   int64 placement_group_final_bundle_placement_timestamp_ms = 15;
 }
+
+message JobTableData {
+  // The job ID.
+  bytes job_id = 1;
+  // Whether it's dead.
+  bool is_dead = 2;
+  // The UNIX timestamp corresponding to this event (job added or removed).
+  int64 timestamp = 3;
+  // IP address of the driver that started this job.
+  // Deprecated
+  string driver_ip_address = 4;
+  // Process ID of the driver running this job.
+  int64 driver_pid = 5;
+  // The config of this job.
+  JobConfig config = 6;
+  // The timestamp the job was started at.
+  uint64 start_time = 7;
+  // The timestamp the job was ended at.
+  uint64 end_time = 8;
+  // The entrypoint of the job.
+  string entrypoint = 9;
+  // Whether this job has running tasks.
+  bool is_running_tasks = 10;
+  // The optional JobInfo from the Ray Job API.
+  optional JobsAPIInfo job_info = 11;
```

Comment:
it was 10 before we broke it? 

https://github.com/ray-project/ray/pull/35188/files#diff-66e7c4b1cbb0a6ce505e17964332b0e4d26631f291bcbabd221d13fa09707d90L391

Reply:
Oh great catch, forgot about that.  Should be fixed now